org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -2624,7 +2624,33 @@ protected void enableExportButton()
 		
 		exportButton.setEnabled(contentPanel.getRowCount() > 0);		
 	}
-	
+
+	/**
+	 * Return (if exists) the editor with the name
+	 * @param columnName
+	 * @return editor
+	 */
+	public WEditor getEditor(String columnName) {
+		for (WEditor editor : editors) {
+			if (editor.getGridField() != null && editor.getGridField().getColumnName().equals(columnName))
+				return editor;
+		}
+		return null;
+	}
+
+	/**
+	 * Return (if exists) the index of the column in the grid
+	 * @param columnName
+	 * @return editor
+	 */
+	public int getColumnIdx(String columnName) {
```

Comment:
should just use getColumnIndex since it is not lengthy

Reply:
done